Controller/Adminhtml/Index/Create.php

```diff
@@ -92,6 +92,10 @@ public function execute()
                 It needs to be installed on a live website. Your website needs to be public and not hidden by a
                 site under construction plugin because it needs connection to our server in order to work.');
                 return $this->jsonResponse(['error' => true, 'data' => []]);
+            } elseif ($this->serverHelper->isServerEnvironment() && !$this->serverHelper->isSslEnabled()) {
```

Comment:
`$this->serverHelper->isServerEnvironment() &&`
Should not be required, as there is only local or server environments.